### PR TITLE
Issue 12 feat/UI create signin signup screen

### PR DESCRIPTION
**Pull request by T0ffik**

### CODE REVIEW

**Review by maciejopalinski — CHANGES_REQUESTED** [review]
I added form handling with React Hook Form and API integration. Please add components for displaying errors to user.

File: src/screens/Register/Register.tsx [hunk 1]
```diff
@@ -118,67 +140,80 @@ export const Register = () => {
...
-                        label='Birthday'
+                        label='Birthdate'
                         type='date'
                         sx={{
                             label: {
                                 display: 'none'
-                            },
-                            '.css-12wo0e5-MuiFormLabel-root-MuiInputLabel-root.Mui-focused':
-                                {
-                                    display: 'block'
```
Comment (T0ffik) [inline src/screens/Register/Register.tsx]: without this style input on focus look like this, so I think that this should stay
image

Comment (maciejopalinski) [reply src/screens/Register/Register.tsx]: it shouldn't be done like that
material ui css selectors are random at each build, find another way of doing it

maybe just leave it for now

**Comment by maciejopalinski** [comment]
ok im done lets finish this task

**Review by maciejopalinski — APPROVED** [review]

**Review by Mariusz-web — APPROVED** [review]

**Pull request merged**